app/Http/Controllers/OrderController.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Cart;
+use App\Models\Order;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Auth;
+
+class OrderController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        //
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $user = Auth::user();
+        $order = new Order();
+
+        $order->user_id = $user->id;
+        $order->total = $request->total;
+        $order->status = 0;
+
+        $order->save();
+
+        $carts = Cart::where('user_id', $user->id)->get();
+
+
+        foreach ($carts as $book) {
+            $book->delete();
+        }
+        return redirect('/carts');
```

Comment:
```suggestion
        DB::table('carts')->whereIn('id', $carts->pluck('id'))->delete();

        return redirect('/carts');
```
giảm số query, add space trc return